stylus/components/forms.styl

```diff
@@ -384,6 +384,15 @@ $select
     &::-ms-expand
         display  none
 
+$select--tiny
+    @extend $input-text--tiny
+
+$select--medium
+    @extend $input-text--medium
+
+$select--fullwidth
+    @extend $input-text--fullwidth
```

Comment:
Why do we need the extra level of indirection ? Why can't the select directly extends from input-text--fullwidth ?